## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 408823`
Commit: `23e414043af4e95956d3e16546aef6bbf7fa1c40`

---
### `x86_64-linux`
<details>
  <summary>:x: 97 packages failed to build:</summary>
  <ul>
    <li>aider-chat-full</li>
    <li>aider-chat-full.dist</li>
    <li>aider-chat-with-help</li>
    <li>aider-chat-with-help.dist</li>
    <li>frigate</li>
    <li>home-assistant-component-tests.openai_conversation</li>
    <li>newelle</li>
    <li>private-gpt</li>
    <li>private-gpt.dist</li>
    <li>python312Packages.llama-cloud-services</li>
    <li>python312Packages.llama-cloud-services.dist</li>
    <li>python312Packages.llama-index</li>
    <li>python312Packages.llama-index-agent-openai</li>
    <li>python312Packages.llama-index-agent-openai.dist</li>
    <li>python312Packages.llama-index-cli</li>
    <li>python312Packages.llama-index-cli.dist</li>
    <li>python312Packages.llama-index-core</li>
    <li>python312Packages.llama-index-core.dist</li>
    <li>python312Packages.llama-index-embeddings-gemini</li>
    <li>python312Packages.llama-index-embeddings-gemini.dist</li>
    <li>python312Packages.llama-index-embeddings-google</li>
    <li>python312Packages.llama-index-embeddings-google.dist</li>
    <li>python312Packages.llama-index-embeddings-huggingface</li>
    <li>python312Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python312Packages.llama-index-embeddings-ollama</li>
    <li>python312Packages.llama-index-embeddings-ollama.dist</li>
    <li>python312Packages.llama-index-embeddings-openai</li>
    <li>python312Packages.llama-index-embeddings-openai.dist</li>
    <li>python312Packages.llama-index-graph-stores-nebula</li>
    <li>python312Packages.llama-index-graph-stores-nebula.dist</li>
    <li>python312Packages.llama-index-graph-stores-neo4j</li>
    <li>python312Packages.llama-index-graph-stores-neo4j.dist</li>
    <li>python312Packages.llama-index-graph-stores-neptune</li>
    <li>python312Packages.llama-index-graph-stores-neptune.dist</li>
    <li>python312Packages.llama-index-indices-managed-llama-cloud</li>
    <li>python312Packages.llama-index-indices-managed-llama-cloud.dist</li>
    <li>python312Packages.llama-index-legacy</li>
    <li>python312Packages.llama-index-legacy.dist</li>
    <li>python312Packages.llama-index-llms-ollama</li>
    <li>python312Packages.llama-index-llms-ollama.dist</li>
    <li>python312Packages.llama-index-llms-openai</li>
    <li>python312Packages.llama-index-llms-openai-like</li>
    <li>python312Packages.llama-index-llms-openai-like.dist</li>
    <li>python312Packages.llama-index-llms-openai.dist</li>
    <li>python312Packages.llama-index-multi-modal-llms-openai</li>
    <li>python312Packages.llama-index-multi-modal-llms-openai.dist</li>
    <li>python312Packages.llama-index-program-openai</li>
    <li>python312Packages.llama-index-program-openai.dist</li>
    <li>python312Packages.llama-index-question-gen-openai</li>
    <li>python312Packages.llama-index-question-gen-openai.dist</li>
    <li>python312Packages.llama-index-readers-database</li>
    <li>python312Packages.llama-index-readers-database.dist</li>
    <li>python312Packages.llama-index-readers-file</li>
    <li>python312Packages.llama-index-readers-file.dist</li>
    <li>python312Packages.llama-index-readers-json</li>
    <li>python312Packages.llama-index-readers-json.dist</li>
    <li>python312Packages.llama-index-readers-llama-parse</li>
    <li>python312Packages.llama-index-readers-llama-parse.dist</li>
    <li>python312Packages.llama-index-readers-s3</li>
    <li>python312Packages.llama-index-readers-s3.dist</li>
    <li>python312Packages.llama-index-readers-twitter</li>
    <li>python312Packages.llama-index-readers-twitter.dist</li>
    <li>python312Packages.llama-index-readers-txtai</li>
    <li>python312Packages.llama-index-readers-txtai.dist</li>
    <li>python312Packages.llama-index-readers-weather</li>
    <li>python312Packages.llama-index-readers-weather.dist</li>
    <li>python312Packages.llama-index-vector-stores-chroma</li>
    <li>python312Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python312Packages.llama-index-vector-stores-google</li>
    <li>python312Packages.llama-index-vector-stores-google.dist</li>
    <li>python312Packages.llama-index-vector-stores-postgres</li>
    <li>python312Packages.llama-index-vector-stores-postgres.dist</li>
    <li>python312Packages.llama-index-vector-stores-qdrant</li>
    <li>python312Packages.llama-index-vector-stores-qdrant.dist</li>
    <li>python312Packages.llama-index.dist</li>
    <li>python312Packages.llama-parse</li>
    <li>python312Packages.llama-parse.dist</li>
    <li>python312Packages.private-gpt</li>
    <li>python312Packages.private-gpt.dist</li>
    <li>python313Packages.llama-index</li>
    <li>python313Packages.llama-index-agent-openai</li>
    <li>python313Packages.llama-index-agent-openai.dist</li>
    <li>python313Packages.llama-index-cli</li>
    <li>python313Packages.llama-index-cli.dist</li>
    <li>python313Packages.llama-index-llms-openai</li>
    <li>python313Packages.llama-index-llms-openai-like</li>
    <li>python313Packages.llama-index-llms-openai-like.dist</li>
    <li>python313Packages.llama-index-llms-openai.dist</li>
    <li>python313Packages.llama-index-multi-modal-llms-openai</li>
    <li>python313Packages.llama-index-multi-modal-llms-openai.dist</li>
    <li>python313Packages.llama-index-program-openai</li>
    <li>python313Packages.llama-index-program-openai.dist</li>
    <li>python313Packages.llama-index-question-gen-openai</li>
    <li>python313Packages.llama-index-question-gen-openai.dist</li>
    <li>python313Packages.llama-index.dist</li>
    <li>python313Packages.private-gpt</li>
    <li>python313Packages.private-gpt.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 439 packages built:</summary>
  <ul>
    <li>aider-chat (python312Packages.aider-chat)</li>
    <li>aider-chat-with-bedrock</li>
    <li>aider-chat-with-bedrock.dist</li>
    <li>aider-chat-with-browser</li>
    <li>aider-chat-with-browser.dist</li>
    <li>aider-chat-with-playwright</li>
    <li>aider-chat-with-playwright.dist</li>
    <li>aider-chat.dist (python312Packages.aider-chat.dist)</li>
    <li>alpaca</li>
    <li>chatblade</li>
    <li>chatblade.dist</li>
    <li>checkov</li>
    <li>checkov.dist</li>
    <li>gnome-frog</li>
    <li>gpt-cli</li>
    <li>gpt-cli.dist</li>
    <li>imaginer</li>
    <li>lacus</li>
    <li>lacus.dist</li>
    <li>lasuite-docs</li>
    <li>lasuite-docs.dist</li>
    <li>litellm</li>
    <li>litellm.dist</li>
    <li>llm</li>
    <li>mealie</li>
    <li>mealie.dist</li>
    <li>mlflow-server</li>
    <li>mlflow-server.dist</li>
    <li>open-interpreter (python312Packages.open-interpreter)</li>
    <li>open-interpreter.dist (python312Packages.open-interpreter.dist)</li>
    <li>open-webui</li>
    <li>open-webui.dist</li>
    <li>openai (python312Packages.openai)</li>
    <li>openai.dist (python312Packages.openai.dist)</li>
    <li>pentestgpt</li>
    <li>pentestgpt.dist</li>
    <li>python312Packages.banks</li>
    <li>python312Packages.banks.dist</li>
    <li>python312Packages.chromadb</li>
    <li>python312Packages.chromadb.dist</li>
    <li>python312Packages.chromadb_0</li>
    <li>python312Packages.chromadb_0.dist</li>
    <li>python312Packages.databricks-sdk</li>
    <li>python312Packages.databricks-sdk.dist</li>
    <li>python312Packages.fnllm</li>
    <li>python312Packages.fnllm.dist</li>
    <li>python312Packages.great-tables</li>
    <li>python312Packages.great-tables.dist</li>
    <li>python312Packages.instructor</li>
    <li>python312Packages.instructor.dist</li>
    <li>python312Packages.jupysql</li>
    <li>python312Packages.jupysql-plugin</li>
    <li>python312Packages.jupysql-plugin.dist</li>
    <li>python312Packages.jupysql.dist</li>
    <li>python312Packages.kserve</li>
    <li>python312Packages.kserve.dist</li>
    <li>python312Packages.lacuscore</li>
    <li>python312Packages.lacuscore.dist</li>
    <li>python312Packages.langchain</li>
    <li>python312Packages.langchain-anthropic</li>
    <li>python312Packages.langchain-anthropic.dist</li>
    <li>python312Packages.langchain-aws</li>
    <li>python312Packages.langchain-aws.dist</li>
    <li>python312Packages.langchain-azure-dynamic-sessions</li>
    <li>python312Packages.langchain-azure-dynamic-sessions.dist</li>
    <li>python312Packages.langchain-chroma</li>
    <li>python312Packages.langchain-chroma.dist</li>
    <li>python312Packages.langchain-community</li>
    <li>python312Packages.langchain-community.dist</li>
    <li>python312Packages.langchain-core</li>
    <li>python312Packages.langchain-core.dist</li>
    <li>python312Packages.langchain-deepseek</li>
    <li>python312Packages.langchain-deepseek.dist</li>
    <li>python312Packages.langchain-fireworks</li>
    <li>python312Packages.langchain-fireworks.dist</li>
    <li>python312Packages.langchain-groq</li>
    <li>python312Packages.langchain-groq.dist</li>
    <li>python312Packages.langchain-huggingface</li>
    <li>python312Packages.langchain-huggingface.dist</li>
    <li>python312Packages.langchain-mistralai</li>
    <li>python312Packages.langchain-mistralai.dist</li>
    <li>python312Packages.langchain-mongodb</li>
    <li>python312Packages.langchain-mongodb.dist</li>
    <li>python312Packages.langchain-ollama</li>
    <li>python312Packages.langchain-ollama.dist</li>
    <li>python312Packages.langchain-openai</li>
    <li>python312Packages.langchain-openai.dist</li>
    <li>python312Packages.langchain-perplexity</li>
    <li>python312Packages.langchain-perplexity.dist</li>
    <li>python312Packages.langchain-tests</li>
    <li>python312Packages.langchain-tests.dist</li>
    <li>python312Packages.langchain-text-splitters</li>
    <li>python312Packages.langchain-text-splitters.dist</li>
    <li>python312Packages.langchain-xai</li>
    <li>python312Packages.langchain-xai.dist</li>
    <li>python312Packages.langchain.dist</li>
    <li>python312Packages.langgraph</li>
    <li>python312Packages.langgraph-checkpoint</li>
    <li>python312Packages.langgraph-checkpoint-postgres</li>
    <li>python312Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python312Packages.langgraph-checkpoint-sqlite</li>
    <li>python312Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python312Packages.langgraph-checkpoint.dist</li>
    <li>python312Packages.langgraph-prebuilt</li>
    <li>python312Packages.langgraph-prebuilt.dist</li>
    <li>python312Packages.langgraph.dist</li>
    <li>python312Packages.langsmith</li>
    <li>python312Packages.langsmith.dist</li>
    <li>python312Packages.lida</li>
    <li>python312Packages.lida.dist</li>
    <li>python312Packages.litellm</li>
    <li>python312Packages.litellm.dist</li>
    <li>python312Packages.llm</li>
    <li>python312Packages.llm-anthropic</li>
    <li>python312Packages.llm-anthropic.dist</li>
    <li>python312Packages.llm-cmd</li>
    <li>python312Packages.llm-cmd.dist</li>
    <li>python312Packages.llm-command-r</li>
    <li>python312Packages.llm-command-r.dist</li>
    <li>python312Packages.llm-deepseek</li>
    <li>python312Packages.llm-deepseek.dist</li>
    <li>python312Packages.llm-docs</li>
    <li>python312Packages.llm-docs.dist</li>
    <li>python312Packages.llm-echo</li>
    <li>python312Packages.llm-echo.dist</li>
    <li>python312Packages.llm-fragments-github</li>
    <li>python312Packages.llm-fragments-github.dist</li>
    <li>python312Packages.llm-fragments-pypi</li>
    <li>python312Packages.llm-fragments-pypi.dist</li>
    <li>python312Packages.llm-fragments-reader</li>
    <li>python312Packages.llm-fragments-reader.dist</li>
    <li>python312Packages.llm-fragments-symbex</li>
    <li>python312Packages.llm-fragments-symbex.dist</li>
    <li>python312Packages.llm-gemini</li>
    <li>python312Packages.llm-gemini.dist</li>
    <li>python312Packages.llm-gguf</li>
    <li>python312Packages.llm-gguf.dist</li>
    <li>python312Packages.llm-git</li>
    <li>python312Packages.llm-git.dist</li>
    <li>python312Packages.llm-grok</li>
    <li>python312Packages.llm-grok.dist</li>
    <li>python312Packages.llm-groq</li>
    <li>python312Packages.llm-groq.dist</li>
    <li>python312Packages.llm-hacker-news</li>
    <li>python312Packages.llm-hacker-news.dist</li>
    <li>python312Packages.llm-jq</li>
    <li>python312Packages.llm-jq.dist</li>
    <li>python312Packages.llm-llama-server</li>
    <li>python312Packages.llm-llama-server.dist</li>
    <li>python312Packages.llm-mistral</li>
    <li>python312Packages.llm-mistral.dist</li>
    <li>python312Packages.llm-ollama</li>
    <li>python312Packages.llm-ollama.dist</li>
    <li>python312Packages.llm-openai-plugin</li>
    <li>python312Packages.llm-openai-plugin.dist</li>
    <li>python312Packages.llm-openrouter</li>
    <li>python312Packages.llm-openrouter.dist</li>
    <li>python312Packages.llm-pdf-to-images</li>
    <li>python312Packages.llm-pdf-to-images.dist</li>
    <li>python312Packages.llm-sentence-transformers</li>
    <li>python312Packages.llm-sentence-transformers.dist</li>
    <li>python312Packages.llm-templates-fabric</li>
    <li>python312Packages.llm-templates-fabric.dist</li>
    <li>python312Packages.llm-templates-github</li>
    <li>python312Packages.llm-templates-github.dist</li>
    <li>python312Packages.llm-tools-datasette</li>
    <li>python312Packages.llm-tools-datasette.dist</li>
    <li>python312Packages.llm-tools-quickjs</li>
    <li>python312Packages.llm-tools-quickjs.dist</li>
    <li>python312Packages.llm-tools-simpleeval</li>
    <li>python312Packages.llm-tools-simpleeval.dist</li>
    <li>python312Packages.llm-tools-sqlite</li>
    <li>python312Packages.llm-tools-sqlite.dist</li>
    <li>python312Packages.llm-venice</li>
    <li>python312Packages.llm-venice.dist</li>
    <li>python312Packages.llm-video-frames</li>
    <li>python312Packages.llm-video-frames.dist</li>
    <li>python312Packages.llm.dist</li>
    <li>python312Packages.llmx</li>
    <li>python312Packages.llmx.dist</li>
    <li>python312Packages.markitdown</li>
    <li>python312Packages.markitdown.dist</li>
    <li>python312Packages.mlflow</li>
    <li>python312Packages.mlflow.dist</li>
    <li>python312Packages.mmcv</li>
    <li>python312Packages.mmcv.dist</li>
    <li>python312Packages.mmengine</li>
    <li>python312Packages.mmengine.dist</li>
    <li>python312Packages.ploomber-core</li>
    <li>python312Packages.ploomber-core.dist</li>
    <li>python312Packages.ploomber-extension</li>
    <li>python312Packages.ploomber-extension.dist</li>
    <li>python312Packages.posthog</li>
    <li>python312Packages.posthog.dist</li>
    <li>python312Packages.sagemaker-mlflow</li>
    <li>python312Packages.sagemaker-mlflow.dist</li>
    <li>python312Packages.shiny</li>
    <li>python312Packages.shiny.dist</li>
    <li>python312Packages.smolagents</li>
    <li>python312Packages.smolagents.dist</li>
    <li>python312Packages.speechrecognition</li>
    <li>python312Packages.speechrecognition.dist</li>
    <li>python312Packages.txtai</li>
    <li>python312Packages.txtai.dist</li>
    <li>vllm (python312Packages.vllm)</li>
    <li>vllm.dist (python312Packages.vllm.dist)</li>
    <li>python313Packages.banks</li>
    <li>python313Packages.banks.dist</li>
    <li>python313Packages.chromadb</li>
    <li>python313Packages.chromadb.dist</li>
    <li>python313Packages.chromadb_0</li>
    <li>python313Packages.chromadb_0.dist</li>
    <li>python313Packages.databricks-sdk</li>
    <li>python313Packages.databricks-sdk.dist</li>
    <li>python313Packages.fnllm</li>
    <li>python313Packages.fnllm.dist</li>
    <li>python313Packages.great-tables</li>
    <li>python313Packages.great-tables.dist</li>
    <li>python313Packages.instructor</li>
    <li>python313Packages.instructor.dist</li>
    <li>python313Packages.jupysql</li>
    <li>python313Packages.jupysql-plugin</li>
    <li>python313Packages.jupysql-plugin.dist</li>
    <li>python313Packages.jupysql.dist</li>
    <li>python313Packages.kserve</li>
    <li>python313Packages.kserve.dist</li>
    <li>python313Packages.lacuscore</li>
    <li>python313Packages.lacuscore.dist</li>
    <li>python313Packages.langchain</li>
    <li>python313Packages.langchain-anthropic</li>
    <li>python313Packages.langchain-anthropic.dist</li>
    <li>python313Packages.langchain-aws</li>
    <li>python313Packages.langchain-aws.dist</li>
    <li>python313Packages.langchain-azure-dynamic-sessions</li>
    <li>python313Packages.langchain-azure-dynamic-sessions.dist</li>
    <li>python313Packages.langchain-chroma</li>
    <li>python313Packages.langchain-chroma.dist</li>
    <li>python313Packages.langchain-community</li>
    <li>python313Packages.langchain-community.dist</li>
    <li>python313Packages.langchain-core</li>
    <li>python313Packages.langchain-core.dist</li>
    <li>python313Packages.langchain-deepseek</li>
    <li>python313Packages.langchain-deepseek.dist</li>
    <li>python313Packages.langchain-fireworks</li>
    <li>python313Packages.langchain-fireworks.dist</li>
    <li>python313Packages.langchain-groq</li>
    <li>python313Packages.langchain-groq.dist</li>
    <li>python313Packages.langchain-huggingface</li>
    <li>python313Packages.langchain-huggingface.dist</li>
    <li>python313Packages.langchain-mistralai</li>
    <li>python313Packages.langchain-mistralai.dist</li>
    <li>python313Packages.langchain-mongodb</li>
    <li>python313Packages.langchain-mongodb.dist</li>
    <li>python313Packages.langchain-ollama</li>
    <li>python313Packages.langchain-ollama.dist</li>
    <li>python313Packages.langchain-openai</li>
    <li>python313Packages.langchain-openai.dist</li>
    <li>python313Packages.langchain-perplexity</li>
    <li>python313Packages.langchain-perplexity.dist</li>
    <li>python313Packages.langchain-tests</li>
    <li>python313Packages.langchain-tests.dist</li>
    <li>python313Packages.langchain-text-splitters</li>
    <li>python313Packages.langchain-text-splitters.dist</li>
    <li>python313Packages.langchain-xai</li>
    <li>python313Packages.langchain-xai.dist</li>
    <li>python313Packages.langchain.dist</li>
    <li>python313Packages.langgraph-checkpoint</li>
    <li>python313Packages.langgraph-checkpoint-postgres</li>
    <li>python313Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python313Packages.langgraph-checkpoint-sqlite</li>
    <li>python313Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python313Packages.langgraph-checkpoint.dist</li>
    <li>python313Packages.langgraph-prebuilt</li>
    <li>python313Packages.langgraph-prebuilt.dist</li>
    <li>python313Packages.langsmith</li>
    <li>python313Packages.langsmith.dist</li>
    <li>python313Packages.lida</li>
    <li>python313Packages.lida.dist</li>
    <li>python313Packages.litellm</li>
    <li>python313Packages.litellm.dist</li>
    <li>python313Packages.llama-cloud-services</li>
    <li>python313Packages.llama-cloud-services.dist</li>
    <li>python313Packages.llama-index-core</li>
    <li>python313Packages.llama-index-core.dist</li>
    <li>python313Packages.llama-index-embeddings-gemini</li>
    <li>python313Packages.llama-index-embeddings-gemini.dist</li>
    <li>python313Packages.llama-index-embeddings-google</li>
    <li>python313Packages.llama-index-embeddings-google.dist</li>
    <li>python313Packages.llama-index-embeddings-huggingface</li>
    <li>python313Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python313Packages.llama-index-embeddings-ollama</li>
    <li>python313Packages.llama-index-embeddings-ollama.dist</li>
    <li>python313Packages.llama-index-embeddings-openai</li>
    <li>python313Packages.llama-index-embeddings-openai.dist</li>
    <li>python313Packages.llama-index-graph-stores-neo4j</li>
    <li>python313Packages.llama-index-graph-stores-neo4j.dist</li>
    <li>python313Packages.llama-index-graph-stores-neptune</li>
    <li>python313Packages.llama-index-graph-stores-neptune.dist</li>
    <li>python313Packages.llama-index-indices-managed-llama-cloud</li>
    <li>python313Packages.llama-index-indices-managed-llama-cloud.dist</li>
    <li>python313Packages.llama-index-legacy</li>
    <li>python313Packages.llama-index-legacy.dist</li>
    <li>python313Packages.llama-index-llms-ollama</li>
    <li>python313Packages.llama-index-llms-ollama.dist</li>
    <li>python313Packages.llama-index-readers-database</li>
    <li>python313Packages.llama-index-readers-database.dist</li>
    <li>python313Packages.llama-index-readers-file</li>
    <li>python313Packages.llama-index-readers-file.dist</li>
    <li>python313Packages.llama-index-readers-json</li>
    <li>python313Packages.llama-index-readers-json.dist</li>
    <li>python313Packages.llama-index-readers-llama-parse</li>
    <li>python313Packages.llama-index-readers-llama-parse.dist</li>
    <li>python313Packages.llama-index-readers-s3</li>
    <li>python313Packages.llama-index-readers-s3.dist</li>
    <li>python313Packages.llama-index-readers-twitter</li>
    <li>python313Packages.llama-index-readers-twitter.dist</li>
    <li>python313Packages.llama-index-readers-txtai</li>
    <li>python313Packages.llama-index-readers-txtai.dist</li>
    <li>python313Packages.llama-index-readers-weather</li>
    <li>python313Packages.llama-index-readers-weather.dist</li>
    <li>python313Packages.llama-index-vector-stores-chroma</li>
    <li>python313Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python313Packages.llama-index-vector-stores-google</li>
    <li>python313Packages.llama-index-vector-stores-google.dist</li>
    <li>python313Packages.llama-index-vector-stores-postgres</li>
    <li>python313Packages.llama-index-vector-stores-postgres.dist</li>
    <li>python313Packages.llama-index-vector-stores-qdrant</li>
    <li>python313Packages.llama-index-vector-stores-qdrant.dist</li>
    <li>python313Packages.llama-parse</li>
    <li>python313Packages.llama-parse.dist</li>
    <li>python313Packages.llm</li>
    <li>python313Packages.llm-anthropic</li>
    <li>python313Packages.llm-anthropic.dist</li>
    <li>python313Packages.llm-cmd</li>
    <li>python313Packages.llm-cmd.dist</li>
    <li>python313Packages.llm-command-r</li>
    <li>python313Packages.llm-command-r.dist</li>
    <li>python313Packages.llm-deepseek</li>
    <li>python313Packages.llm-deepseek.dist</li>
    <li>python313Packages.llm-docs</li>
    <li>python313Packages.llm-docs.dist</li>
    <li>python313Packages.llm-echo</li>
    <li>python313Packages.llm-echo.dist</li>
    <li>python313Packages.llm-fragments-github</li>
    <li>python313Packages.llm-fragments-github.dist</li>
    <li>python313Packages.llm-fragments-pypi</li>
    <li>python313Packages.llm-fragments-pypi.dist</li>
    <li>python313Packages.llm-fragments-reader</li>
    <li>python313Packages.llm-fragments-reader.dist</li>
    <li>python313Packages.llm-fragments-symbex</li>
    <li>python313Packages.llm-fragments-symbex.dist</li>
    <li>python313Packages.llm-gemini</li>
    <li>python313Packages.llm-gemini.dist</li>
    <li>python313Packages.llm-gguf</li>
    <li>python313Packages.llm-gguf.dist</li>
    <li>python313Packages.llm-git</li>
    <li>python313Packages.llm-git.dist</li>
    <li>python313Packages.llm-grok</li>
    <li>python313Packages.llm-grok.dist</li>
    <li>python313Packages.llm-groq</li>
    <li>python313Packages.llm-groq.dist</li>
    <li>python313Packages.llm-hacker-news</li>
    <li>python313Packages.llm-hacker-news.dist</li>
    <li>python313Packages.llm-jq</li>
    <li>python313Packages.llm-jq.dist</li>
    <li>python313Packages.llm-llama-server</li>
    <li>python313Packages.llm-llama-server.dist</li>
    <li>python313Packages.llm-mistral</li>
    <li>python313Packages.llm-mistral.dist</li>
    <li>python313Packages.llm-ollama</li>
    <li>python313Packages.llm-ollama.dist</li>
    <li>python313Packages.llm-openai-plugin</li>
    <li>python313Packages.llm-openai-plugin.dist</li>
    <li>python313Packages.llm-openrouter</li>
    <li>python313Packages.llm-openrouter.dist</li>
    <li>python313Packages.llm-pdf-to-images</li>
    <li>python313Packages.llm-pdf-to-images.dist</li>
    <li>python313Packages.llm-sentence-transformers</li>
    <li>python313Packages.llm-sentence-transformers.dist</li>
    <li>python313Packages.llm-templates-fabric</li>
    <li>python313Packages.llm-templates-fabric.dist</li>
    <li>python313Packages.llm-templates-github</li>
    <li>python313Packages.llm-templates-github.dist</li>
    <li>python313Packages.llm-tools-datasette</li>
    <li>python313Packages.llm-tools-datasette.dist</li>
    <li>python313Packages.llm-tools-quickjs</li>
    <li>python313Packages.llm-tools-quickjs.dist</li>
    <li>python313Packages.llm-tools-simpleeval</li>
    <li>python313Packages.llm-tools-simpleeval.dist</li>
    <li>python313Packages.llm-tools-sqlite</li>
    <li>python313Packages.llm-tools-sqlite.dist</li>
    <li>python313Packages.llm-venice</li>
    <li>python313Packages.llm-venice.dist</li>
    <li>python313Packages.llm-video-frames</li>
    <li>python313Packages.llm-video-frames.dist</li>
    <li>python313Packages.llm.dist</li>
    <li>python313Packages.llmx</li>
    <li>python313Packages.llmx.dist</li>
    <li>python313Packages.markitdown</li>
    <li>python313Packages.markitdown.dist</li>
    <li>python313Packages.mlflow</li>
    <li>python313Packages.mlflow.dist</li>
    <li>python313Packages.open-interpreter</li>
    <li>python313Packages.open-interpreter.dist</li>
    <li>python313Packages.openai</li>
    <li>python313Packages.openai.dist</li>
    <li>python313Packages.ploomber-core</li>
    <li>python313Packages.ploomber-core.dist</li>
    <li>python313Packages.ploomber-extension</li>
    <li>python313Packages.ploomber-extension.dist</li>
    <li>python313Packages.posthog</li>
    <li>python313Packages.posthog.dist</li>
    <li>python313Packages.sagemaker-mlflow</li>
    <li>python313Packages.sagemaker-mlflow.dist</li>
    <li>python313Packages.shiny</li>
    <li>python313Packages.shiny.dist</li>
    <li>python313Packages.smolagents</li>
    <li>python313Packages.smolagents.dist</li>
    <li>python313Packages.speechrecognition</li>
    <li>python313Packages.speechrecognition.dist</li>
    <li>python313Packages.txtai</li>
    <li>python313Packages.txtai.dist</li>
    <li>python313Packages.vllm</li>
    <li>python313Packages.vllm.dist</li>
    <li>seagoat</li>
    <li>seagoat.dist</li>
    <li>shell-genie</li>
    <li>shell-genie.dist</li>
    <li>shell-gpt</li>
    <li>shell-gpt.dist</li>
    <li>sploitscan</li>
    <li>sploitscan.dist</li>
    <li>textlsp</li>
    <li>textlsp.dist</li>
    <li>tftui</li>
    <li>tftui.dist</li>
    <li>vectorcode</li>
    <li>vectorcode.dist</li>
    <li>vimPlugins.vectorcode-nvim</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>frigate</summary>
<pre>frigate/test/test_object_detector.py::TestLocalObjectDetector::test_detect_raw_given_tensor_input_should_return_api_detect_raw_result
  /build/frigate-0.15.1-source/frigate/test/test_object_detector.py:47: PydanticDeprecatedSince20: `parse_obj_as` is deprecated. Use `pydantic.TypeAdapter.validate_python` instead. Deprecated in Pydantic V2.0 to be removed in V3.0. See Pydantic V2 Migration Guide at https://errors.pydantic.dev/2.11/migration/
    detector_config=parse_obj_as(DetectorConfig, {"type": "cpu", "model": {}})

frigate/test/test_object_detector.py::TestLocalObjectDetector::test_localdetectorprocess_should_only_create_specified_detector_type
frigate/test/test_object_detector.py::TestLocalObjectDetector::test_localdetectorprocess_should_only_create_specified_detector_type
frigate/test/test_object_detector.py::TestLocalObjectDetector::test_localdetectorprocess_should_only_create_specified_detector_type
frigate/test/test_object_detector.py::TestLocalObjectDetector::test_localdetectorprocess_should_only_create_specified_detector_type
frigate/test/test_object_detector.py::TestLocalObjectDetector::test_localdetectorprocess_should_only_create_specified_detector_type
frigate/test/test_object_detector.py::TestLocalObjectDetector::test_localdetectorprocess_should_only_create_specified_detector_type
frigate/test/test_object_detector.py::TestLocalObjectDetector::test_localdetectorprocess_should_only_create_specified_detector_type
frigate/test/test_object_detector.py::TestLocalObjectDetector::test_localdetectorprocess_should_only_create_specified_detector_type
frigate/test/test_object_detector.py::TestLocalObjectDetector::test_localdetectorprocess_should_only_create_specified_detector_type
  /build/frigate-0.15.1-source/frigate/test/test_object_detector.py:22: PydanticDeprecatedSince20: `parse_obj_as` is deprecated. Use `pydantic.TypeAdapter.validate_python` instead. Deprecated in Pydantic V2.0 to be removed in V3.0. See Pydantic V2 Migration Guide at https://errors.pydantic.dev/2.11/migration/
    test_cfg = parse_obj_as(

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED frigate/test/test_http.py::TestHttp::test_recordings - assert []
==== 1 failed, 165 passed, 1 deselected, 1837 warnings in 190.28s (0:03:10) ====</pre>
</details>
<details>
<summary>home-assistant-component-tests.openai_conversation</summary>
<pre>        __loader__ = <_pytest.assertion.rewrite.AssertionRewritingHook object at 0x7ffff6987a10>
        __name__   = 'tests.components.openai_conversation.test_conversation'
        __package__ = 'tests.components.openai_conversation'
        __spec__   = ModuleSpec(name='tests.components.openai_conversation.test_conversation', loader=<_pytest.assertion.rewrite.AssertionRewritingHook object at 0x7ffff6987a10>, origin='/build/source/tests/components/openai_conversation/test_conversation.py')
        async_expose_entity = <function async_expose_entity at 0x7ffff025b1a0>
        async_setup_component = <function async_setup_component at 0x7ffff2bd3b00>
        conversation = <module 'homeassistant.components.conversation' from '/build/source/homeassistant/components/conversation/__init__.py'>
        httpx      = <module 'httpx' from '/nix/store/k899c4j4l4hnf5rmjzfz7qx40b4h0vqn-python3.13-httpx-0.28.1/lib/python3.13/site-packages/httpx/__init__.py'>
        intent     = <module 'homeassistant.helpers.intent' from '/build/source/homeassistant/helpers/intent.py'>
        mock_chat_log = <function mock_chat_log at 0x7fffee907a60>
        mock_create_stream = <function mock_create_stream at 0x7fffee9076a0>
        patch      = <function patch at 0x7ffff560f6a0>
        pytest     = <module 'pytest' from '/nix/store/hkfyg5vls2farazb6vvvaqmnzdzi79xm-python3.13-pytest-8.3.5/lib/python3.13/site-packages/pytest/__init__.py'>
        test_entity = <function test_entity at 0x7fffee928180>
        test_error_handling = <function test_error_handling at 0x7fffee9282c0>
        test_incomplete_response = <function test_incomplete_response at 0x7fffee928360>
=========================== short test summary info ============================
ERROR tests/components/openai_conversation/test_conversation.py - pydantic_core._pydantic_core.ValidationError: 1 validation error for Respon...
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
=============================== 1 error in 1.33s ===============================</pre>
</details>
<details>
<summary>python313Packages.llama-index-llms-openai</summary>
<pre>                                  ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<string>", line 1, in <lambda>
    import sys; import importlib; list(map(lambda mod: importlib.import_module(mod), sys.argv[1:]))
                                                       ~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "/nix/store/2mab9iiwhcqwk75qwvp3zv0bvbiaq6cs-python3-3.13.3/lib/python3.13/importlib/__init__.py", line 88, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 1026, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/nix/store/1c4nc1sysnq56qi9cwygb36sprz57l7x-python3.13-llama-index-llms-openai-0.3.42/lib/python3.13/site-packages/llama_index/llms/openai/__init__.py", line 2, in <module>
    from llama_index.llms.openai.responses import OpenAIResponses
  File "/nix/store/1c4nc1sysnq56qi9cwygb36sprz57l7x-python3.13-llama-index-llms-openai-0.3.42/lib/python3.13/site-packages/llama_index/llms/openai/responses.py", line 6, in <module>
    from openai.types.responses import (
    ...<19 lines>...
    )
ImportError: cannot import name 'ResponseTextAnnotationDeltaEvent' from 'openai.types.responses' (/nix/store/85sq3bj90895x5vnyfbwdj7wicl12am4-python3.13-openai-1.86.0/lib/python3.13/site-packages/openai/types/responses/__init__.py)</pre>
</details>
</details>

--- a/pkgs/development/python-modules/openai/default.nix
+++ b/pkgs/development/python-modules/openai/default.nix
@@ -46,7 +46,7 @@
 
 buildPythonPackage rec {
   pname = "openai";
-  version = "1.82.0";
+  version = "1.86.0";
   pyproject = true;
 
   disabled = pythonOlder "3.8";
@@ -55,7 +55,7 @@ buildPythonPackage rec {
     owner = "openai";
     repo = "openai-python";
     tag = "v${version}";
-    hash = "sha256-BN78ZTtBTTEbr46KZCOFc00IQN7jA1rCwn8Xv+iZ0r8=";
+    hash = "sha256-PDYyuvCkDfQrbkSz0CPfJr++WUu5mODY2nVzTanwqjo=";
   };
 
   postPatch = ''substituteInPlace pyproject.toml --replace-fail "hatchling==1.26.3" "hatchling"'';

